### PR TITLE
[hotfix] clean special chars util

### PR DESCRIPTION
**Pull request by EdgarCordova**
case: https://greenchef.atlassian.net/browse/WTF-644

### CODE REVIEW

File: middleware/clean-text.middleware.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const latinize = require('latinize');
+
+module.exports = (string) => {
+  let latinizedText = latinize(string);
```
Comment (bearfact) [inline middleware/clean-text.middleware.js:4]: might have some tabs vs spaces here.. it looks funny

File: middleware/index.js [hunk 2]
```diff
@@ -1,13 +1,15 @@
 const asyncWrap = require('./async.middleware');
+const cleanText = require('./clean-text.middleware');
```
Comment (bearfact) [inline middleware/index.js:2]: This isn't middleware.  It needs to go in a new "utils" directory

**Review by bearfact — APPROVED** [review]

**Review by svet93 — APPROVED** [review]

**Comment by svet93** [comment]
👍

**Comment by bearfact** [comment]
:+1:

**Comment by EdgarCordova** [comment]
👍

**Pull request merged**